src/Surfnet/ServiceProviderDashboard/Application/CommandHandler/Entity/UpdateEntityIdpsCommandHandler.php

```diff
@@ -58,7 +58,7 @@ public function handle(UpdateEntityIdpsCommand $command): void
         $allowedIdps = new AllowedIdentityProviders($idps, $allowedAll);
         $entity->getAllowedIdentityProviders()->merge($allowedIdps);
         try {
-            $this->publishClient->publish($entity, $entity, 'ACL');
+            $this->publishClient->publish($entity, $entity, $command->applicant, 'ACL');
```

Comment:
Just a thought: if you use named parameters, the param list can be in any order.

Reply:
Yup, I know about them, but I'm still reluctant in using them. I guess I'm not comfortable with them yet.